data-dictionary.yaml

```diff
@@ -0,0 +1,110 @@
+
+    Note that the 'Value sets' and `Value set codes' sheets do not provide any information about value sets that are defined outside of mCODE.
+
+    **Example:** 'Secondary Cancer Condition Code' is a data element contained by the 'Secondary Cancer Condition' profile. The 'Secondary Cancer Condition Code' data element is associated with the <http://hl7.org/fhir/us/mcode/ValueSet/mcode-secondary-cancer-disorder-vs> value set in the 'Data elements' sheet, which indicates the value set is defined in mCODE. Additional information on this value set can be found in the 'Value sets' and 'Value set details' sheet; the former includes a description of the value set, and the latter lists the specific codes and descriptions as well as logical definitions for this value set.
+
+  '"Value sets" sheet': |+
+    The 'Value Sets' sheet contains a list of mCODE-defined value sets, as well as high-level information about them.  Externally-defined value sets used by mCODE, such as LOINC answer lists and value sets defined in US Core or FHIR, are not included.
+
+    **Relationship to 'Value Set Details' sheet:** In general, the 'Value Set Details' sheet contains enumerated codes and associated descriptions for a given value set, or a logical definition (when a value set is defined by a rule, rather than an explicit list of codes).
+
+    **Example:** The 'CancerConditionVS' is comprised of both enumerated ICD-10-CM codes, and a logical definition for SNOMED CT codes: "includes all codes descending from includes codes descending from 108369006 'Neoplasm (morphologic abnormality)."
+
+  '"Value set codes" sheet': |+
+    The 'Value set codes' sheet contains individual codes associated with each value set defined by mCODE. Externally-defined value sets are not included.
+
+    Note that not every code that belongs in an mCODE value set may be included in the 'Value set codes' sheet. There are several reasons for this: 1) The value set is logically defined; 2) Intellectual property restrictions associated with certain code systems (e.g. AJCC, ICD-O-3, CPT); or 3) The codes for a particular code system have not been formally restricted due to technical limitations in the tool that automatically manages the publication of the mCODE FHIR Implementation Guide.
+
+  COLUMN KEY: ''
+
+  Profiles: |+
+    **Group**: The general area of mCODE, either Patient, Disease, Genomics, Treatment, or Outcome. For an overview of mCODE domains, consult the diagram included in the mCODE FHIR Implementation Guide.
+
+    **Title**: The name of the profile.
+
+    **Url**: Web address identifying the profile.
```

Comment:
Is this the canonical URL or a real web address? Or a URL that redirects to the actual web address?